rero_ils/modules/patrons/templates/rero_ils/_macro_profile.html

```diff
@@ -70,7 +70,7 @@ <h6>{{ _('Details') }}</h6>
               {% endif %}
               <div class="row">
                   <div class="col-lg-2 col-sm-2 label-title">
-                    <b>{{ _('Loan started at') }}</b>
+                    <b>{{ _('Loan started') }}</b>
```

Comment:
Did you see the comment on this string on weblate : https://hosted.weblate.org/translate/rero_plus/rero-ils/fr/?checksum=9ebc20bb300f3158 ?